doc/whats_new/upcoming_changes/sklearn.utils/30149.enhancement.rst

```diff
@@ -0,0 +1,8 @@
+- :func:`~utils.estimator_checks.check_estimator` now accepts the list of tests that are
```

Comment:
This really doesn't feel right as a single chengelog entry @lesteve

Reply:
:sweat: :shrug: :sweat:

If we think this is crucial, here are the work-arounds I can think of:
- do nested bullet points for each aspect
- use multiple bullet points and mention the `:pr:`30149` in each of them except the last bullet point, since towncrier will add the 

To double-check the generated rst file:
```
towncrier build --version 1.6.0 --draft
```

Reply:
We also need to add this to our doc build, so that we can see the rendered version of the changelog on the CI